File: frontend/src/employer.ts
import {User} from './User'
export class Employer extends User{
    regName:string;
    field:string;
    city:string;
    state:string;
    country:string;
    pincode:number;
    description:string;
    webaddress:string;
    points:number;
    placementPortalUrl:string;

    /*  {
        "username": "xyz",
        "password": "258",
        "emailId": "xyz@gmail",
        "role": "employer",
        "status": "true",
        "regName": "alfred",
        "field": "petrolum",
        "city": "jiddah",
        "state": "saudi",
        "country": "saudi",
        "pincode": 41412,
        "description": "petroleum production company ",
        "webAddress": "www.al",
        "points": 0,
        "placemetPortalUrl": null,
        "empFeedback": null,
        "id": 2
    },*/ 

}

File: backend/BOL/Employer.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace BOL
{
    public class Employer:User
    {
        public string regName { get; set; }
        public string field { get; set; }
        public string city { get; set; }
        public string state { get; set; }
        public string country { get; set; }
        public int pinCode { get; set; }
        public string description { get; set; }
        public string webAddress { get; set; }
        public int points { get; set; }
        public string placementPortalUrl { get; set; }
    }
}
